test/unit-tests/yok.ts

```diff
@@ -601,9 +601,12 @@ describe("yok", () => {
 	});
 
 	function deleteDirectory(directory: string): boolean {
-		shelljs.rm("-rf", directory);
-		let err = shelljs.error();
-		return err === null;
+		try {
+			shelljs.rm("-rf", directory);
```

Comment:
I think here `shelljs.config.fatal` will be `false`. Can you check and if I am right we must keep the old logic.